lib/ui/view/channel/edit_channel/edit_channel_view_model.dart

```diff
@@ -18,15 +20,23 @@ class EditChannelViewModel extends FormViewModel {
   final storage = locator<SharedPreferenceLocalStorage>();
   final snackbar = locator<SnackbarService>();
   final _apiService = ZuriApi(channelsBaseUrl);
+  final _navigationService = locator<NavigationService>();
+  String id = '';
   String? get token => storage.getString(StorageKeys.currentSessionToken);
   bool isLoading = false;
   loading(status) {
     isLoading = status;
     notifyListeners();
   }
 
+  onChanged(String val) {
+    // log.i(val);
+    notifyListeners();
```

Comment:
Since there is nothing to notify, then remove this line

Reply:
Removed!